src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -107,6 +112,51 @@ public static class MachineOptions {
                 "This does nothing if B:enableCleanroom is false.",
                 "Default: false"})
         public boolean cleanMultiblocks = false;
+
+        public class LargeMinerOptions {
```

Comment:
Don't give the player full control over everything
This will allow player to play the mod in not intended way
In this case, you only need largeMinerMace and you should set the max value for @Config.RangeInt

(I saw many PRs got rejected because they give player too much options)